examples/src/4_origins/mod.rs

```diff
@@ -5,17 +5,20 @@ mod tests {
 	use xcm::latest::prelude::*;
 	use xcm_simulator::TestExt;
 
-	const AMOUNT: u128 = 10;
 	const QUERY_ID: u64 = 1234;
 
 	/// Scenario:
 	#[test]
 	fn descend_origin() {
 		MockNet::reset();
 		ParaA::execute_with(|| {
+			let message_fee = parachain::estimate_message_fee(6);
 			let message = Xcm(vec![
-				WithdrawAsset((Here, AMOUNT).into()),
-				BuyExecution { fees: (Here, AMOUNT).into(), weight_limit: WeightLimit::Unlimited },
+				WithdrawAsset((Here, message_fee).into()),
+				BuyExecution {
+					fees: (Here, message_fee).into(),
+					weight_limit: WeightLimit::Unlimited,
```

Comment:
We could limit this with `estimate_weight`

Reply:
We could do it, but then I propose we do it in a separate pr for all examples